core/src/main/scala/kafka/admin/ConsumerGroupCommand.scala

```diff
@@ -54,8 +54,13 @@ object ConsumerGroupCommand {
     opts.checkArgs()
 
     val consumerGroupService = {
-      if (opts.options.has(opts.newConsumerOpt)) new KafkaConsumerGroupService(opts)
-      else new ZkConsumerGroupService(opts)
+      if (opts.useOldConsumer) {
+        Console.err.println("Using the ConsumerGroupCommand with old consumer is deprecated and will be removed in a future release. " +
+          "Consider using the new consumer.")
```

Comment:
I don't think we should deprecate this. As I said in my comment: "We should not deprecate this because the old consumer is not deprecated and if they are using ZK-based consumer groups, this is the only way to get the required information."